php/test/test_async.php

```diff
@@ -113,12 +113,12 @@ function json_stringify($s) {
 }
 
 function convert_to_snake_case($input) {
-    return strtolower(preg_replace('/(?<!^)(?=[A-Z])/', '_', $input));
+    $res = strtolower(preg_replace('/(?<!^)(?=[A-Z])/', '_', $input));
+    return str_replace('o_h_l_c_v', 'ohlcv', $res);
```

Comment:
this method had been used in different methods, but only some of them had "inline-fix" for ohlcv. so i've moved to here central plaec